### DISCUSSION TITLE
Generated kWh

### DISCUSSION THREAD

**Discussion by io-taster** [softwarecrash/EPEver2MQTT#24] [General]
Die Epever laufen bereits einige Monate.
Heute am 1.9. ist mir aufgefallen das die Tagesleistung von der Monatsleistung abweicht. 
Das sollte am ersten Tag des Monats nicht sein.
Bzw. die Tagesleistung wurde bereits um 17:00 zurück gesetzt. Datum und Uhrzeit passt. Siehe Bild.
![20230901_173905](https://github.com/softwarecrash/EPEver2MQTT/assets/124024833/5ca4b4a1-98b0-4caf-8885-3ef3ca4c426c)
![20230901_174450](https://github.com/softwarecrash/EPEver2MQTT/assets/124024833/f197b415-751b-4a5f-9413-079f7543f09f)

**Comment by all-solutions** [comment]
Da musst Du den EPEver "treten". Wir fragen nur Werte ab, der Wemos berechnet da nichts :wink:
